components/local-app/pkg/helper/workspace.go

```diff
@@ -200,13 +201,13 @@ func ObserveWorkspaceUntilStarted(ctx context.Context, clnt *client.Gitpod, work
 				return ws, nil
 			}
 
-			var currentStatus string
 			if HasInstanceStatus(wsInfo.Msg.Result) {
-				currentStatus = prettyprint.FormatWorkspacePhase(wsInfo.Msg.Result.Status.Instance.Status.Phase)
```

Comment:
This actually made it so that the status never got updates (should be `ws` instead of `wsInfo`). Here we fix it as well